crypto/cms/cms_asn1.c

```diff
@@ -245,6 +245,16 @@ ASN1_NDEF_SEQUENCE(CMS_EncryptedData) = {
         ASN1_IMP_SET_OF_OPT(CMS_EncryptedData, unprotectedAttrs, X509_ATTRIBUTE, 1)
 } ASN1_NDEF_SEQUENCE_END(CMS_EncryptedData)
 
+ASN1_NDEF_SEQUENCE(CMS_AuthEnvelopedData) = {
```

Comment:
Add a reference to the RFC documentation and section that this is defined in..

Reply:
Do you mean just adding a comment. Something like `Defined in RFC 5084`?

Reply:
it's actually RFC 5083 :)

Reply:
Yes.. But a little more details .. e.g RFC XXX "Section Blah : TitleX"

Reply:
Pushed - please let me know if the format is fine?